app/views/install/compose.phtml

```diff
@@ -583,25 +622,37 @@ services:
       - _APP_REDIS_PORT
       - _APP_REDIS_USER
       - _APP_REDIS_PASS
-      - _APP_DB_HOST	
-      - _APP_DB_PORT	
-      - _APP_DB_SCHEMA	
-      - _APP_DB_USER	
+      - _APP_DB_HOST
+      - _APP_DB_PORT
+      - _APP_DB_SCHEMA
+      - _APP_DB_USER
       - _APP_DB_PASS
 
+  appwrite-assistant:
+    image: appwrite/assistant:0.2.0
```

Comment:
I think this should be 0.2.1

Reply:
0.2.1 added dotenv which should only be used for dev so we're okay without it.

Reply:
yea we can

Reply:
Isn't it required to read the API key from the env?